app/controllers/sidebars_controller.rb

```diff
@@ -11,7 +11,7 @@ def show
 
   def get_latest_campaign_articles
     campaign_articles_scope = Article.tagged_with(Campaign.current.featured_tags, any: true)
-      .where("published_at > ? AND score > ?", 4.weeks.ago, 0)
+      .where("published_at > ? AND score > ?", SiteConfig.campaign_articles_expiry_time.weeks.ago, 0)
```

Comment:
**suggestion (non-blocking):** A part of me wants to be really explicit here and call the config option `campaign_article_expiry_in_weeks`.